### DISCUSSION TITLE
Yubikey for wikijs ?

### DISCUSSION THREAD

**Discussion by hansolo57** [requarks/wiki#5590] [Help / Questions] [answered]
Hello, 

The documentation says that Fido2 is supported. https://docs.requarks.io/auth

I would like to use a yubikey for 2FA to login. How to use it with Wikijs ? Do we need to use an external product as Keycloak for fido 2 ?

Best regards

**Comment by NGPixel** [comment, marked as answer]
The docs says that it is planned for a future release, not that it's supported. You cannot use a yubikey at the moment.
